src/main/scala/scala/tools/refactoring/util/SourceWithMarker.scala

```diff
@@ -274,16 +335,16 @@ object SourceWithMarker {
   }
 
   object Movement {
-    def apply(impl: (SourceWithMarker, Boolean) => Option[Int]): Movement = {
+    def apply(impl: (SourceWithMarker, Boolean) => Seq[Int]): Movement = {
```

Comment:
would be fine to write what `apply` takes as argument in `trait Movement` (especially what is in returned `Seq` / `Option` ) (better Scaladoc)